modules/elasticsearch/src/main/java/org/testcontainers/elasticsearch/ElasticsearchContainer.java

```diff
@@ -0,0 +1,90 @@
+/*
```

Comment:
Are you the copyright holder? WDYT about relicensing under MIT, to be in line with the rest of Testcontainers? It's not a must, but might be easier in the long run for the users.

Reply:
Ha! That's because I migrated my code to your repo but forgot to change the header. I'll update it.

Reply:
Hmmm. Looking at the Couchbase header for example, I can see that it's under an Apache2 License with Couchbase as the copyright holder. I guess this should be changed at some point?

Reply:
MIT license doesn't request to have license headers in the actual source files. AFAIK it's okay to just have the LICENSE file in the project (but IANAL).